### ISSUE TITLE
Error with dev-env database import

### ISSUE THREAD

**Issue by dinova-dtardia** [Automattic/vip-cli#1210]
## Actual Behavior

Using latest version v2.4.1 the import command for dev-env give node error and search and replace is not executed. After that is not possible to call import again.
Also downgrading to prior version I need to destroy and re-create dev-env to import database 

## Steps to Reproduce the Problem

- First call
  - Command
  ```
  :$ vip dev-env import sql ./***.sql --skip-validate \
  --search-replace="vt_***,wordpress" \
  --search-replace="https://***,http://***.vipdev.lndo.site" \
  --search-replace="***,***.vipdev.lndo.site" \
  --slug ***
  ```
  - Console message
  ```
  Running validation steps...
  ✓ Check for docker installation 
  ✓ Check for docker-compose installation 
  ✓ Check access to docker for current user 
  ✓ Check DNS resolution 
  Success: Imported from '/app/***.sql'.
  Success: The cache was flushed.
  (node:85421) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 65 post-init listeners added to [AsyncEvents]. Use emitter.setMaxListeners() to increase limit
  (Use `node --trace-warnings ...` to show where the warning was created)
  (node:85421) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 65 post-start listeners added to [AsyncEvents]. Use emitter.setMaxListeners() to increase limit
  User "vipgo" already exits. Skipping creating it.`
  ```

- Second call
  - Command
  ```
  :$ vip dev-env import sql ./***.sql --skip-validate \
  --search-replace="vt_***,wordpress" \
  --search-replace="https://***,http://***.vipdev.lndo.site" \
  --search-replace="***,***.vipdev.lndo.site" \
  --slug ***
  ```
  - Console message
  ```
  Running validation steps...
  ✓ Check for docker installation 
  ✓ Check for docker-compose installation 
  ✓ Check access to docker for current user 
  ✓ Check DNS resolution 
  Error: Import file missing or not readable: /user/.local/share/vip/dev-environment/***/***.sql
  Error: Error: Import file missing or not readable: /user/.local/share/vip/dev-environment/***/***.sql
  ```
Thank you

**Comment by sjinks** [comment]
Could you please try 2.24.2?

**Comment by dinova-dtardia** [comment]
Hi @sjinks I've just try with version 2.24.2 and I have same errors
```
(node:75911) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 65 post-init listeners added to [AsyncEvents]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
(node:75911) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 65 post-start listeners added to [AsyncEvents]. Use emitter.setMaxListeners() to increase limit
```
but import result seems to be ok

**Comment by sjinks** [comment]
The `MaxListenersExceededWarning` warning will be fixed in 2.25 (it is [already in the `develop` branch](https://github.com/Automattic/vip-cli/blob/d6c98ad847314198eabd93b68656c366f5a67ac3/src/bin/vip-dev-env-import-sql.js#L58)) but not yet released; the proper fix was introduced in #1200.

Those warnings should not affect the import.

**Comment by sjinks** [comment]
Since the import works, I am closing this issue then...
